enterprise/users/src/test/java/io/crate/integrationtests/UserManagementIntegrationTest.java

```diff
@@ -130,4 +130,10 @@ public void testDropSuperUserThrowsException() throws Exception {
         expectedException.expectMessage("UnsupportedFeatureException: Cannot drop a superuser 'crate'");
         executeAsSuperuser("drop user crate");
     }
+
+    @Test
+    public void testCreateUserWithPassword() throws Exception {
+        executeAsSuperuser("create user foo with (password = 'password')");
+        assertUserIsCreated("foo");
```

Comment:
Can we check the password as well?

Reply:
hm. difficult since the generated hash is not predictable / deterministic.

Reply:
I agree with @mxm, we should verify here that the user can execute a query with the defined password.

Reply:
there will be other tests with the last PR
https://github.com/crate/crate/commit/d71db3731be03f8a1037f21271173ec27364f117#diff-0293163e51773d21ddef26a112a9eaa2

Reply:
@mikethebeer For a full integration test, testing if you can authenticate with the newly created user would be nice.

Reply:
@seut as @chaudum said this will be possible with the last PR. This PR only enhances the statement and the storage/migration of user-meta data.